app/views/request-psp-test-account/index.njk

```diff
@@ -30,20 +33,21 @@
               You can download the data as a CSV file or by integrating with our reporting and reconciliation API.
             </p>
             <p class="govuk-body">
-              We can usually set up a Stripe test account for your service within 1 working day. We'll email you when it's ready and include information about making demo payments and accessing reports.
-            </p>
-            <p class="govuk-body">
-              You'll find the Stripe test account in Overview under the service name. You'll have both a Pay test account ("sandbox") and Stripe test account.
+              If you choose to get a Stripe test account, it will replace your sandbox account
+              immediately. You will not be able to change your account back. If you have any existing API keys they
+              will be disabled.
             </p>
             <p class="govuk-body">
-              If you’re not ready to set up a test account, you can
-               <a class="govuk-link" href="https://www.payments.service.gov.uk/payment-service-provider/">learn more about Stripe and how it works</a>.
+              <span class="govuk-!-font-weight-bold">You will need to create new API keys and add them to your
+              integration as your old keys will stop working.</span>
```

Comment:
just questioning the content here, this only applies to users of the Pay API, but it will show in bold to all users. This might make some service users think they can't proceed as they don't understand the requirement for keys.

Reply:
Looking at the [proposed content](https://docs.google.com/document/d/1v42sT2rrMqWuRQ8K0JZx1OjfmsTtDuLu4R41XiTt5sc/edit#heading=h.lrrd9jrf1t71) this shouldn't have been in bold so I've fixed it.